bin/node/runtime/src/lib.rs

```diff
@@ -2166,4 +2166,35 @@ mod tests {
 			size,
 		);
 	}
+
+	#[test]
+	fn check_pallet_storage_sizes() {
+		use frame_support::traits::StorageInfoTrait;
+		let mut storage_info = AllPalletsWithSystem::storage_info();
+		println!("| Pallet | Storage | Max Values | Max Size |");
+		println!("| --- | --- | --- | --- |");
+
+		storage_info.sort_by_key(|k| k.max_size);
+		storage_info.reverse();
+
+		let mut failed = 0;
+
+		for info in storage_info {
+			let pallet_name = String::from_utf8(info.pallet_name).unwrap();
+			let storage_name = String::from_utf8(info.storage_name).unwrap();
+			println!(
+				"| {:?} | {:?} | {:?} | {:?} |",
+				pallet_name, storage_name, info.max_values, info.max_size
+			);
```

Comment:
```suggestion
            println!(
				"| {:<30} | {:<30} | {:<10} | {:<15} |",
				pallet_name,
				storage_name,
				format!("{:?}", info.max_values),
				format!("{:?}", info.max_size)
			);
```

Which leaves us with this formatting:
```
|             Pallet             |            Storage             | Max Values |    Max Size     |
| ------------------------------ | ------------------------------ | ---------- | --------------- |
| Contracts                      | DeletionQueue                  | Some(1)    | Some(20537014)  |
| ImOnline                       | ReceivedHeartbeats             | None       | Some(10021032)  |
| Preimage                       | PreimageFor                    | None       | Some(4194340)   |
| Contracts                      | CodeStorage                    | None       | Some(393358)    |
| ImOnline                       | Keys                           | Some(1)    | Some(320002)    |
```